base/math.jl

```diff
@@ -709,18 +722,18 @@ according to the rounding mode `r`. In other words, the quantity
 without any intermediate rounding.
 
 - if `r == RoundNearest`, then the result is exact, and in the interval
-  ``[-|y|/2, |y|/2]``.
+  ``[-|y|/2, |y|/2]``. See [`RoundNearest`](@ref) for more.
```

Comment:
For more what? May I suggest
```
See also [`RoundNearest`](@ref).
```
instead, which is what we have in most other places I think.

Reply:
ok :)